test/Dialect/FIRRTL/annotations.fir

```diff
@@ -44,12 +44,12 @@ circuit Foo: %[[{"a":"a","target":"Foo.Foo"}]]
 ; // -----
 
 ; A ModuleTarget Annotation is attached to the correct module.
-circuit Foo: %[[{"a":"a","target":"~Foo|Foo"}]]
-  module Foo:
+circuit FooMt: %[[{"a":"a","target":"~FooMt|FooMt"}]]
+  module FooMt:
```

Comment:
I understand why this is rewriting all the test names, but I like my canonical metasyntactic names. 👹 I won't hold this up for that, but if it's easy you can land these on `main` separately go for it.